.NET/Microsoft.Recognizers.Text.DateTime/Extractors/ISetExtractorConfiguration.cs

```diff
@@ -33,5 +35,7 @@ public interface ISetExtractorConfiguration : IOptionsConfiguration
         IDateTimeExtractor TimePeriodExtractor { get; }
 
         IDateTimeExtractor DateTimePeriodExtractor { get; }
+
+        void SetWeekDayExtractor(IDateTimeExtractor extractor, string text, Match match, DateObject reference, ref List<Token> ret);

```

Comment:
Better to use composition instead of inheritance, to avoid most cultures having empty methods.